pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2337,7 +2336,7 @@ public class ServiceConfiguration implements PulsarConfiguration {
         category = CATEGORY_LOAD_BALANCER,
         doc = "Supported algorithms name for namespace bundle split"
     )
-    private List<String> supportedNamespaceBundleSplitAlgorithms = Lists.newArrayList("range_equally_divide",
+    private Set<String> supportedNamespaceBundleSplitAlgorithms = Sets.newHashSet("range_equally_divide",
```

Comment:
@coderzc are you on an old base branch? @MarvinCai IIRC we enable modernize plugin for `pulsar-broker-common` module and this usage should be banned.

Reply:
No, I base on master. See:
https://github.com/apache/pulsar/blob/master/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java#L2335-L2341

I should be change it to `Set.of()` ?

Reply:
I'd prefer to do so since #16991 is ongoing. @youzipi & @MarvinCai may investigate why there's still such remaining.

Reply:
I found still keep a lot `Sets.newHashSet(...)` in the https://github.com/apache/pulsar/issues/16991.

Reply:
> create an unmodifiable set

That's it! I think JDK provides only the unmodifiable set version and `new HashSet<>()` which doesn't accept an element parameter. Thanks for your investigation!